### PR TITLE
Disable test input causing Corelib assertion

### PR DESCRIPTION
**Pull request by RussKeldorph**
See #13238

### CODE REVIEW

File: src/System.Reflection.Emit/tests/ModuleBuilder/ModuleBuilderDefineEnum.cs [hunk 1]
```diff
@@ -14,7 +14,7 @@ public class ModuleBuilderDefineEnum
 
         public static IEnumerable<object[]> DefineEnum_TestData()
         {
-            foreach (string name in new string[] { "TestEnum", "testenum", "enum", "\uD800\uDC00", "a\0b\0c" })
+            foreach (string name in new string[] { "TestEnum", "testenum", "enum", "\uD800\uDC00" /*, "a\0b\0c" */})
```
Comment (stephentoub) [inline src/System.Reflection.Emit/tests/ModuleBuilder/ModuleBuilderDefineEnum.cs]: Please add a comment with the associated coreclr issue.

**Comment by RussKeldorph** [comment]
Thanks for the help @stephentoub.  Is that the right location for the comment, or should it be on the method?

**Comment by stephentoub** [comment]
LGTM. Thanks, @RussKeldorph.

**Comment by stephentoub** [comment]
@dotnet-bot Test Innerloop Windows_NT Release Build and Tes please ("The "VerifyDependencies" task failed unexpectedly.")

**Comment by stephentoub** [comment]
@dotnet-bot Test Innerloop Windows_NT Release Build and Test please ("The "VerifyDependencies" task failed unexpectedly.")

**Pull request merged**